checkov/arm/runner.py

```diff
@@ -26,9 +28,10 @@ def run(
         root_folder: str,
         external_checks_dir: Optional[List[str]] = None,
         files: Optional[List[str]] = None,
-        runner_filter: RunnerFilter = RunnerFilter(),
+        runner_filter: RunnerFilter | None = None,
```

Comment:
why? flake8?

Reply:
yes, correct. a function call as an argument default value will be invoked once even when you invoke it again. It is not so critical here, because we only invoke the `run` method once per a whole run, but why take the risk. Instead use `None` and if it is not passed as an argument it will be instantiated a few lines lower.